data/campaigns/Winds_of_Fate/scenarios/10_Fire_Meets_Steel.cfg

```diff
@@ -0,0 +1,815 @@
+        {MODIFY_UNIT id=$unit.id facing se}
+        {CLEAR_VARIABLE available_holes,random}
+    [/event]
+    # End of dwarf hole events
+
+    [event]
+        name=prestart
+        {MODIFY_UNIT side=1 facing se}
+        {MODIFY_UNIT side=2 facing se}
+        [objectives]
+            [objective]
+                description= _ "Outlast the intruders for 3 days. (18 Turns)"
+                condition=win
+            [/objective]
+            [objective]
+                description= _ "Death of Gorlack"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Arinexis"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Zedrix"
+                condition=lose
+            [/objective]
+            [gold_carryover]
+                carryover_percentage=0
+            [/gold_carryover]
+            [note]
+                description= _ "The establishment of our new Eyrie provides bountiful resources. (5 Gold Per Village, Recruitable Veterans)"
+            [/note]
+            [note]
+                description= _ "Deep dwellers are emerging from their burrows deep in the underland."
+            [/note]
+            [note]
+                description= _ "If a drake can reach a burrow, it will destroy it. A saurian might do likewise."
+            [/note]
+            [note]
+                description= _ "It is the opinion of our auxiliary Arinexis, that these creatures will besiege us for three days as a Trial of Domain."
+            [/note]
+        [/objectives]
+        # Prepare for starting dialog
+        {TELEPORT_UNIT id=Gorlack  27 19}
+        {TELEPORT_UNIT id=Zedrix 28 18}
+        [put_to_recall_list]
+            id=Arinexis
+        [/put_to_recall_list]
+        [modify_side]
+            side=1
+            shroud=yes
+        [/modify_side]
+    [/event]
+
+    [event]
+        name=start
+        [message]
+            speaker=Zedrix
+            message= _ "I sssee now you are not Shek’kahan. I ssaw as I wanted to ssee."
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "As have I..."
+        [/message]
+        [message]
+            speaker=Zedrix
+            message= _ "What do you mean?"
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "As a fledgling long ago...
+I strayed far from the eyrie of vladnir, lost in dreams of glorious things to be.
+
+From a cave nearby I heard cries.
+My friend Reshan had happened upon a dishonored rogue hiding within.
+It sought his death to keep secret its presence there.
+
+It had cornered him."
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "I thought myself invulnerable, the hero in my own tale.
+I charged at the beast, striking with bare claws.
+Spiting feeble sparks.
```

Comment:
Is this supposed to be 'spitting'? To spite means something else, plus given drakes breathe fire, I assume spitting is correct here.

Reply:
Good catch!  That's one the spellchecker would never find.